src/main/java/reactor/netty/tcp/ProxyProvider.java

```diff
@@ -75,6 +78,7 @@ public static ProxyProvider findProxySupport(Bootstrap b) {
 	final Function<? super String, ? extends String> password;
 	final Supplier<? extends InetSocketAddress> address;
 	final Pattern nonProxyHosts;
+	final Supplier<? extends HttpHeaders> httpHeaders;
```

Comment:
Do we need this as Supplier?

Reply:
Because I would like to lazy-evaluate it, I defined this instance as Supplier. The auth-token header may often has an expiration time.

Reply:
ok thanks